carve.lic

```diff
@@ -175,6 +175,7 @@ class Carve
     end
     waitrt?
     magic_cleanup
+    Flags.delete('carve-magic-ready')
```

Comment:
❓  Should the Flag deletion be part of `magic_cleanup`? (in each of these files)